chart/epinio/templates/cert-secrets.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if not .Values.certManager.enabled }}
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  annotations:
+  name: epinio-registry-tls
+  namespace: {{ .Release.Namespace }}
+stringData:
+  ca.crt: {{ .values.certManager.registry.ca | quote }}
```

Comment:
```suggestion
  ca.crt: {{ .Values.certManager.registry.ca | quote }}
```

Reply:
Please fix the `.Values` capitalization everywhere.

Reply:
Done, commit 40b8ae445e1f6e010ed29f8e32035bc29525a4c2